specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2015-03-20/OperationalInsights.json

```diff
@@ -929,9 +947,9 @@
           "format": "date-time",
           "description": "The time of last update."
         },
-        "eTag": {
+        "etag": {
```

Comment:
This causes breaking changes for a relatively superficial fix. I'd leave it alone and fix it in a future API Version.

Reply:
Ok